deploy/helm/flightctl/values.yaml

```diff
@@ -40,6 +43,13 @@ global:
   internalNamespace: ""
   clusterLevelSecretAccess: false
   appCode: ""
+  exposeServicesMethod: "nodePort" # nodePort, route
```

Comment:
```suggestion
  exposeServicesMethod: "nodePort" # nodePort, route
```

I would make it the other way around: default to route, ... but for kind we set "nodePort"

making it just a tiny bit harder for developers, but cleaner for customers.

Reply:
This would have been easier than all the "acm" defaults to router change ... sorry about that just noticed it now.

Reply:
Ok, the default is `router` and docs are updated too